examples/Azure_IoT_Central_Arduino_Nano_RP2040_Connect/Azure_IoT_PnP_Template.cpp

```diff
@@ -0,0 +1,544 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <stdlib.h>
+#include <stdarg.h>
+
+#include <az_core.h>
+#include <az_iot.h>
+
+#include "AzureIoT.h"
+#include "Azure_IoT_PnP_Template.h"
+
+#include <az_precondition_internal.h>
+
+/* --- Defines --- */
+#define AZURE_PNP_MODEL_ID "dtmi:azureiot:devkit:freertos:Esp32AzureIotKit;1"
+
+#define SAMPLE_DEVICE_INFORMATION_NAME                 "deviceInformation"
+#define SAMPLE_MANUFACTURER_PROPERTY_NAME              "manufacturer"
+#define SAMPLE_MODEL_PROPERTY_NAME                     "model"
+#define SAMPLE_SOFTWARE_VERSION_PROPERTY_NAME          "swVersion"
+#define SAMPLE_OS_NAME_PROPERTY_NAME                   "osName"
+#define SAMPLE_PROCESSOR_ARCHITECTURE_PROPERTY_NAME    "processorArchitecture"
+#define SAMPLE_PROCESSOR_MANUFACTURER_PROPERTY_NAME    "processorManufacturer"
+#define SAMPLE_TOTAL_STORAGE_PROPERTY_NAME             "totalStorage"
+#define SAMPLE_TOTAL_MEMORY_PROPERTY_NAME              "totalMemory"
+
+#define SAMPLE_MANUFACTURER_PROPERTY_VALUE             "Arduino"
+#define SAMPLE_MODEL_PROPERTY_VALUE                    "Arduino Nano RP2040 Connet"
```

Comment:
Do a find all and replace on `Connet` just to double check
```suggestion
#define SAMPLE_MODEL_PROPERTY_VALUE                    "Arduino Nano RP2040 Connect"
```